include/roaring/roaring.h

```diff
@@ -168,7 +168,6 @@ void roaring_bitmap_andnot_inplace(roaring_bitmap_t *x1,
  */
 void roaring_bitmap_free(roaring_bitmap_t *r);
 
-
```

Comment:
apparently I felt the need to move this empty line to roaring.c for some reason! oops